pt/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Resumo de Aprendizagem Profunda
```

Comment:
@gabriel19913 I used `Dicas` as a translation for Cheastsheet. It would be better to set an unique term. What are your thoughts on this?

Reply:
Yes, that makes sense. I think it's a better term.